site/content/docs/5.3/components/footer.md

```diff
@@ -22,6 +22,8 @@ You can choose to display each part or not, except the last one which is mandato
 
 If these available parts don't meet your specific needs, feel free to develop your own custom part accordingly to the Orange Design System.
 
+The footer is positioned at the bottom of the window when the page content is smaller than the window, and at the bottom of the page when it is bigger.
```

Comment:
Might not be true for some DOM reasons. But for simple syntax where `<footer>` is a direct child of `<body>` it seems to work pretty well. Any opinion on this @julien-deramond ?